src/endpoint/schema.ts

```diff
@@ -277,25 +280,34 @@ export class SchemaEndpoint extends Endpoint {
 
 	/**
 	 * Create an ST Schema connector
+	 *
 	 * @param data definition of the connector
+	 * @param organizationId The organization to associate the connector with. You must be a member
+	 * of the organization. Overrides any organization specified
 	 */
-	public create(data: SchemaAppRequest): Promise<SchemaCreateResponse> {
-		return this.client.post<SchemaCreateResponse>('apps', data)
+	public create(data: SchemaAppRequest, organizationId?: string): Promise<SchemaCreateResponse> {
+		const options = organizationId ? { headerOverrides: { 'X-ST-Organization': organizationId } } : undefined
+		return this.client.post<SchemaCreateResponse>('apps', data, undefined, options)
 	}
 
 	/**
 	 * Update an ST Schema connector
+	 *
 	 * @param id the "endpointApp" UUID of the connector, e.g. "viper_799ff3a0-8249-11e9-9bf1-b5c7d651c2c3"
 	 * @param data new definition of the connector
+	 * @param organizationId The organization to associate the connector with. You must be a member
+	 * of the organization. The organization cannot be changed if the schema connector has been certified.
```

Comment:
Since I'm not really sure how we should be wording things, I just decided to be very specific and say

> The organization cannot be changed if the connector's `certificationStatus` is `wwst`.